src/main/resources/MyView.properties

```diff
@@ -0,0 +1,4 @@
+UI_PRIMARY = #e56060
```

Comment:
My_____ are names for tutorials only

Reply:
Refactored to "Application" resource bundle (would have called it "Language" but it's also storing colors right now)